py-polars/polars/internals/expr/expr.py

```diff
@@ -684,6 +697,46 @@ def keep_name(self) -> Self:
         """
         return self._from_pyexpr(self._pyexpr.keep_name())
 
+    def pipe(
+        self,
+        func: Callable[Concatenate[Expr, P], T],
```

Comment:
Please name this parameter `function`. We will do that for other pipe method as well.

```suggestion
        function: Callable[Concatenate[Expr, P], T],
```